### PR TITLE
Add cleanSchedule method by scheSn

### PR DESCRIPTION
**Pull request by chenyang14**

### CODE REVIEW

File: dataAccessObjects.py [hunk 1]
```diff
@@ -152,8 +152,11 @@ def updateEditSchedule(self,targetId,displayTime,modeSn,scheSn):
             +" WHERE sche_sn={scheSn}".format(scheSn=scheSn)
         self.db.cmd(sql)
 
-    def cleanSchedule(self):
-        sql = 'DELETE FROM schedule WHERE sche_is_used=0'
+    def cleanSchedule(self,scheSn=None):
+        if scheSn:
+            sql = "DELETE FROM schedule WHERE sche_sn={scheSn}".format(scheSn=scheSn)
+        else:
+            sql = 'DELETE FROM schedule WHERE sche_is_used=0'
```
Comment (stvreumi) [inline dataAccessObjects.py]: Why don't I see line `159` in the original place?

Comment (chenyang14) [inline dataAccessObjects.py]: Line 159 is the same as line156 in original dataAcessObjects.py

Comment (stvreumi) [inline dataAccessObjects.py]: Sorry...

Comment (chenyang14) [inline dataAccessObjects.py]: Never mind. XD

**Pull request merged**